play/src/pages/gameinprogress/ChooseAnswer.tsx

```diff
@@ -90,31 +130,19 @@ export default function ChooseAnswer({
           selectedAnswer={selectedAnswer}
           handleSelectAnswer={handleSelectAnswer}
         />
+        {displaySubmitted ? onSubmitDisplay : null}
```

Comment:
I think that this behaviour works for the in-progress branch of this overall Confidence feature. However, I think we'll need to coordinate with the Design team to see how they are planning to enable or disable this feature (I assume on the host side at some point). We can talk through this further on paired programming tomorrow

Reply:
Sounds good!